File: src/tests/Markify.Solution.Tests/CSharp.Project/Wellformed/TestDelegate.cs
namespace CSharp.Project.Wellformed
{
    public delegate void TestDelegate();

    public delegate int TestDelegate2();

    public delegate void TestDelegate3(int foo);

    public delegate int TestDelegate4(int foo);

    public delegate void TestDelegate5(params int[] foo);

    public delegate int TestDelegate6(params int[] foo);
}

File: src/tests/Markify.Solution.Tests/VisualBasic.Project/Wellformed/TestDelegate.vb
Namespace CSharp.Project.Wellformed
    Public Delegate Sub TestDelegate()

    Public Delegate Function TestDelegate2() As Integer

    Public Delegate Sub TestDelegate3(foo As Integer)

    Public Delegate Function TestDelegate4(foo As Integer) As Integer

    Public Delegate Sub TestDelegate5(<[ParamArray]()> foo() As Integer)

    Public Delegate Function TestDelegate6(<[ParamArray]()> foo() As Integer) As Integer
End Namespace